lib/iex/test/iex/helpers_test.exs

```diff
@@ -49,6 +49,22 @@ defmodule IEx.HelpersTest do
     assert capture_io(fn -> h __info__ end) == "No documentation for __info__ was found\n"
   end
 
+  test "h helper for callbacks" do
+    with_file ["a_behaviour.ex", "impl.ex"], [behaviour_module, impl_module], fn ->
```

Comment:
Protocols are behaviours... so the implementations are callback implementations... maybe we could use that? Also, let's get rid of the preamble for now. :)